android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -952,6 +1102,12 @@ internal class CommonKtTests {
 
 const val MICROS_MULTIPLIER = 1_000_000
 
+fun getPresentedOfferingId(purchaseParams: PurchaseParams) : String? {
+    // Uses reflection to test presentedOfferingIdentifier exists in purchasing params
+    val prop = PurchaseParams::class.members.firstOrNull { member -> member.name == "presentedOfferingIdentifier" }
+    return prop!!.call(purchaseParams) as? String?
+}
```

Comment:
We don't make `presentedOfferingIdentifier` public on `PurchaseParams` so had to use reflection to test 😬 

Not sure if there is a better way to do this but it worked 🤷‍♂️

Reply:
Well, the other way I was thinking would be mockking the `PurchaseParams` constructor (mockk has a `mockkConstructor` function) to return a `mockk` and we can validate that the methods we expect are called in that mockk. I would prefer that rather than reflection? I try to avoid reflection as much as possible 😬.

Reply:
Just talked with @joshdholtz and it might be a bit tricky to use `mockkConstructor` here. For the sake of shipping and considering this is only a test and we are planning to refactor it in the future. We can go ahead with this.

Reply:
Talked with @tonidero about how to do this because I'm not a `mockk` expert and decided that we are going to keep this reflection test **FOR NOW**.

Will create a new task to expose `presentedOfferingIdentifier` as a getting on `PurchaseParams` which will allow us to remove this reflection test in the future.